README.md

```diff
@@ -529,9 +529,9 @@ This package includes a model called `metricflow_time_spine.sql` that MetricFlow
 vars:
   ad_reporting__metricflow_time_spine_enabled: false ## true by default
 ```
-Additionally, the `dbt_date.get_base_dates` macro is used in the generation of the `metricsflow_time_spine.sql` model. This macro requires the `dbt_date:time_zone` variable to be defined in the project to generate a time spine based on the defined time zone. The default value in this package is `America/Los_Angeles`. However, you may override this variable in your own project if you wish.
+Additionally, the `fivetran_get_base_dates` macro is used in the generation of the `metricsflow_time_spine` model. This macro requires the `dbt_date:time_zone` variable to be defined in the project to generate a time spine based on the defined time zone. The default value in this package is `America/Los_Angeles`. However, you may override this variable in your own project if you wish.
 
->**Note**: This variable is defined under the `ad_reporting` hierarchy within this package and should not adjust any local global variable values in your project if you already have this variable defined. For more information on why this variable is needed and the different value options, refer to the [dbt-date package documentation](https://github.com/calogica/dbt-date#variables).
+>**Note**: This `dbt_date:time_zone` variable is defined under the `ad_reporting` hierarchy within this package and should not adjust any local global variable values in your project if you already have this variable defined. For more information on why this variable is needed and the different value options, refer to the Variable section of the [dbt-date package documentation](https://github.com/godatadriven/dbt-date/tree/main?tab=readme-ov-file#variables).
```

Comment:
This is a bit strange that we have decided to host the macros, but then still point to a different repo to explain the variable. The variable section we link to here is quite small. Let's just recreate it here instead so we aren't referencing a package we aren't actually using.

Reply:
Made some updates, let me know what you think